src/Catalog/AppendOnlyCatalogWriter.cs

```diff
@@ -47,18 +47,20 @@ protected override Uri[] GetAdditionalRootType()
             bool isExistingPage;
             Uri pageUri = GetPageUri(pageEntries, itemEntries.Count, out isExistingPage);
 
+            IDictionary<string, CatalogItemSummary> items = new Dictionary<string, CatalogItemSummary>(itemEntries);
```

Comment:
fixed

---
In reply to: [57126881](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57126881) [](ancestors = 57126881)